src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  health = 100;
```

Comment:
Example of using a constant:
```suggestion
  health = MAX_HEALTH;
```